content/docs/tech-insights/add-scorecard/index.md

```diff
@@ -6,15 +6,49 @@ description: Adding a scorecard.
 
 Once you click the “Add Scorecard” button, you’ll be prompted to fill in the details about the Scorecard, including the Checks you want to include and to which entities to apply it to, as shown below:
 
-![Create Scorecard](./create-scorecard.png)
+### About section
 
-1 - General information about scorecard (Name and Description).
+![About section](./create-scorecard-about.png)
 
-2 - Checks you wish to include in the scorecard. Clicking ‘Add’ button will lead to opening the dialog.
+- Give a unique name to your Scorecard
+- Give a proper description about this Scorecard. This input accepts markdown
 
-![Add check](./add-check.png)
+### Checks
 
-Within the dialog you select all the checks you want to include. In case you want to add or remove some of the checks later you can do it later ‘Edit’ functionality.
+![Checks](./create-scorecard-add-checks.png)
+
+Click the `Add` button to open the add checks modal
+
+![Add checks modal](./create-scorecard-add-checks-modal.png)
+
+- Select the checks you would like to add to your scorecard
+- Close the modal once you added all of the checks you would want in your scorecard
+
+![Added checks](create-scorecard-added-checks.png)
+
+### Applies to
+
+You can define filters to target which entities should this scorecard run against
+In the `Applies to` section you can select which entites will be included in this scorecard and which entities will be excluded.
+
+![Applies to section](create-scorecard-applies-to.png)
+
+The above configuration means the scorecard will run against components with type library that does not have the tag infrastructure
+
+Providing multiple selection in a single input results in an `OR` relation between the selections.
+Selecting from multiple input fields end up with a relation `AND` between the fields.
+
+For example:
+Selecting `type: service, tags: infrastructure` will exclude every enetity Which has type service `AND` contains an infrastructure tag. However it won't exclude services which does not contain the tag, or components that contain only the tag but they are not services.
```

Comment:
`entity which` rather than `enetity Which`

Reply:
`services which do not contain the tag` rather than `services which does not contain the tag`